.gitignore

```diff
@@ -72,4 +72,5 @@ Cargo.lock
 x/programs/simulator/libsimulator.h
 x/programs/runtime/debug
 x/programs/runtime/wasm32-unknown-unknown
-x/programs/runtime/.rustc_info.json
```

Comment:
Always add newlines at the end of files. I tried adding a new filename here and broke this because there was no new line